wro4j-core/src/main/java/ro/isdc/wro/model/factory/XmlModelFactory.java

```diff
@@ -76,40 +76,40 @@
   /**
    * Group tag used in xml.
    */
-  private static final String TAG_GROUP = "group";
+  protected static final String TAG_GROUP = "group";
   /**
    * Import tag used in xml.
    */
-  private static final String TAG_IMPORT = "import";
+  protected static final String TAG_IMPORT = "import";
   /**
    * CSS tag used in xml.
    */
-  private static final String TAG_CSS = "css";
+  protected static final String TAG_CSS = "css";
 
   /**
    * JS tag used in xml.
    */
-  private static final String TAG_JS = "js";
+  protected static final String TAG_JS = "js";
 
   /**
    * GroupRef tag used in xml.
    */
-  private static final String TAG_GROUP_REF = "group-ref";
+  protected static final String TAG_GROUP_REF = "group-ref";
 
   /**
    * Group name attribute.
    */
-  private static final String ATTR_GROUP_NAME = "name";
+  protected static final String ATTR_GROUP_NAME = "name";
   /**
    * Group abstract attribute used to distinguish an abstract group when its value is true. By default the value is
    * false.
    */
-  private static final String ATTR_GROUP_ABSTRACT = "abstract";
+  protected static final String ATTR_GROUP_ABSTRACT = "abstract";
   /**
    * Minimize attribute specified on resource level, used to turn on/off minimization on this particular resource during
    * pre processing.
    */
-  private static final String ATTR_MINIMIZE = "minimize";
+  protected static final String ATTR_MINIMIZE = "minimize";

```

Comment:
Wondering if it is really necessary to increase visibility of all the constants. Do you see any use-case when the extending class would need to access them?